README.md

```diff
@@ -2,9 +2,10 @@
 
 ![Dhruva Shaw Banner.](https://dhruvacube.github.io/images/Banner.jpg)
 
-## Description
+## All about GradeGiver
 
-This is GradeGiver. Which gives grades according to CBSE CCE system.
+Hey eveyone!
```

Comment:
hey @Shubhi-Paul please change `eveyone` to `everyone` :)

Reply:
oops sorry